content/python/concepts/iterators/terms/iter.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'iter()'
+Description: 'The iter() method is a python built-in function that returns an iterator object for a given argument.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Iterators'
+  - 'OOP'
+  - 'Loops'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+The `iter()` method is a python built-in function that returns an iterator object for a given `object`.
+
+## Syntax
+
+```pseudo
+iter(object, sentinel[optional])
+```
+
+| Parameter  | Description                                                                                                               | 
```

Comment:
This would be better as a bulleted list than a table.

Reply:
I'll change that.
(... was so happy to make markdown table ;-) )

Reply:
I added the requested bullet list